test/uk/gov/hmrc/residencenilratebandcalculator/controllers/TransitionControllerSpec.scala

```diff
@@ -78,6 +81,61 @@ class TransitionControllerSpec extends CommonPlaySpec with MockSessionConnector
       status(result) mustBe SEE_OTHER
       redirectLocation(result) mustBe Some(routes.SessionExpiredController.onPageLoad.url)
     }
+
+    def nonStartingController[A: ClassTag](
+        createController: () => SimpleControllerBase[A],
+        answerRowConstants: List[String]
+    )(rds: Reads[A], wts: Writes[A]): Unit = {
+
+      "On a page load with an expired session, return an redirect to an expired session page" in {
+        expireSessionConnector()
+        val result = createController().onPageLoad(rds)(fakeRequest)
```

Comment:
Please use empty lines between test initiation, method-under-test execution and assertions.